core/lib/spree/core/engine.rb

```diff
@@ -101,6 +101,13 @@ class Engine < ::Rails::Engine
       initializer "spree.core.checking_migrations" do |app|
         Migrations.new(config, engine_name).check
       end
+
+      config.to_prepare do
+        # Load application's model / class decorators
+        Dir.glob(File.join(File.dirname(__FILE__), '../../../app/**/*_decorator*.rb')) do |c|
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.